tests/_data/models.py

```diff
@@ -1047,6 +1048,18 @@ def get_bom_with_multiple_licenses() -> Bom:
     )
 
 
+def get_bom_with_tools_with_component_and_service() -> Bom:
```

Comment:
add function name suffix `_migrate` - since tour serialized data is not able to deserialized to the same model, when coming from CDX < 1.5